_posts/2021-01-16-sr2021-first-league-sessions.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: SR2021 First League Session a Great Success!
+---
+
+After the first 36 matches of the SR2021 Competition, team HRS3 from
+Hills Road Sixth Form College head up the initial league leaderboard with a
+staggering 82 game points. They're closely followed by the other 3 Hills Road
+teams as well as one of our rookie teams We Robots.
+
+The league matches provide a proving ground for all the teams' hard work over
+the past few months in preparing their robot code. There is also a prize for the
+rookie team which scores the best in the league, which both the We Robots team
+and Harris Westminster Sixth Form Team 1 are currently well placed for.
+
+At the end of the league we announced to teams the details of our
+[second game module]({{ site.baseurl }}/docs/resources/2021/rulebook.html).
```

Comment:
Since this is a link outside this repo I would encourage using an absolute link:
```suggestion
[second game module](https://studentrobotics.org/docs/resources/2021/rulebook.html).
```

Reply:
https://github.com/srobo/website/pull/253/commits/6bf86337df3925699da15ba8731fa17c0f1cf514